evmcore/tx_pool_test.go

```diff
@@ -497,9 +497,9 @@ func TestEIP4844Transactions(t *testing.T) {
 		err    error
 	}{
 		{"empty blob tx before cancun", nil, false, ErrTxTypeNotSupported},
-		{"blob tx before cancun", common.Address{1}.Bytes(), false, ErrTxTypeNotSupported},
+		{"blob tx before cancun", common.Address{1}.Bytes(), false, ErrOversizedData},
```

Comment:
The fact that the produced error changed may undermine the intention of this test. Can you update the transaction that is tested here such that it passes the `OversizedData` check and reaches the error that it previously was supposed to produce?

Reply:
https://github.com/0xsoniclabs/sonic/pull/200/commits/2724ab4e13337c83cbe9efe010de5df4c21bf2ba